jobs/uaa/templates/bin/pre-start.erb

```diff
@@ -123,7 +123,7 @@ function process_certs {
 function insert_ssl_cert {
     log "Installing Server SSL certificate"
 
-    openssl pkcs12 -export -name uaa_ssl_cert \
+    openssl pkcs12 -export -certpbe PBE-SHA1-3DES -name uaa_ssl_cert \
```

Comment:
Can we expose this through a bosh property, instead of hard coding this change?

Reply:
you could also check (`cat /proc/sys/crypto/fips_enabled`) if FIPS is enabled and only add that parameter in the FIPS case.

Reply:
https://github.com/cloudfoundry/uaa-release/pull/407 is an alternative using the fix I suggested here.